### PR TITLE
Update docs for Logitech Media Server name change to Lyrion Music Server

### LINKED ISSUE #32217: Squeezebox: Logitech Media Server was renamed to Lyrion Music Server
### Feedback

Logitech Media Server was renamed to Lyrion Music Server as announced on its [website](https://lyrion.org/) while the abbreviation LMS stays the same.
You might want to change the docs and integration accordingly.

### URL

https://www.home-assistant.io/integrations/squeezebox

### Version

2024.4.2

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by peteS-UK**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/119480
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5569
- This PR fixes or closes issue: fixes #32217

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Updated the Squeezebox integration to support the Lyrion Music Server ecosystem.
  
- **Documentation**
  - Changed all references from Logitech Media Server to Lyrion Music Server.
  - Updated descriptions and configurations to reflect the new ecosystem.
  - Ensured compatibility with HTTPS reverse proxies for Lyrion Music Servers.

### CODE REVIEW

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -15,16 +15,18 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Squeezebox integration allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [Squeezelite](https://github.com/ralph-irving/squeezelite), [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
+The Squeezebox integration allow you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was previously known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
```
Comment (erwindouna) [inline source/_integrations/squeezebox.markdown]: Shouldn't this be "allows"?

**Review by erwindouna — CHANGES_REQUESTED** [review]

File: source/_integrations/squeezebox.markdown (hunk 1 above)
Comment (peteS-UK) [reply source/_integrations/squeezebox.markdown]: Yes - sorry.  Do you want me to change it in a new PR or will you change it?

Comment (frenck) [reply source/_integrations/squeezebox.markdown]: @peteS-UK In general, review comments are meant to be solve by the contributor; which is you in this case 😉 

../Frenck

**Comment by peteS-UK** [comment]
That’s fine – just checking that was easiest for you guys.  I’ll submit an update if there are no other comments.

**Comment by erwindouna** [comment]
> That’s fine – just checking that was easiest for you guys.  I’ll submit an update if there are no other comments.
> 

Sure. Feel free to commit the update. :)

File: source/_integrations/squeezebox.markdown (hunk 1 above)
Comment (peteS-UK) [reply source/_integrations/squeezebox.markdown]: OK - update committed :-)

**Review by erwindouna — APPROVED** [review]
This now looks great to me. Thanks peteS!

**Review by frenck — APPROVED** [review]
Thanks, @peteS-UK 👍 

../Frenck

**Pull request merged**